### PR TITLE
[Rust] Added doc comments for Global and GlobalType;

### PR DESCRIPTION
**Pull request by apepkuss**
This PR is to add the necessary descriptions for Global and GlobalType.

### CODE REVIEW

**Comment by apepkuss** [comment]
Hi @hydai, could you please help review this PR? Thanks a lot!

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by apepkuss** [comment]
@hydai The issues have been fixed. Thanks a lot!

**Comment by apepkuss** [comment]
Hi @hydai, could you please help review the latest changes? The review comments have been fixed. Thanks a lot!

File: bindings/rust/wasmedge-sys/src/types.rs [hunk 1]
```diff
@@ -219,3 +221,146 @@ impl fmt::Display for ExternType {
...
+    fn from(f: Box<dyn Fn(Vec<Value>) -> Result<Vec<Value>, u8>>) -> Self {
+        let f_ptr = &f as *const dyn Fn(Vec<Value>) -> Result<Vec<Value>, u8> as *mut c_void;
+        unsafe { wasmedge::WasmEdge_ValueGenExternRef(f_ptr).into() }
+    }
+}
+impl From<Value> for wasmedge::WasmEdge_Value {
+    fn from(value: Value) -> Self {
+        match value {
+            Value::I32(v) => Self {
+                Value: v as u128,
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/types.rs:271]: Why do we map `I32` to `u128`?

File: bindings/rust/wasmedge-sys/src/types.rs [hunk 2]
```diff
@@ -219,3 +221,146 @@ impl fmt::Display for ExternType {
...
+                Type: wasmedge::WasmEdge_ValType_I32,
+            },
+            Value::I64(v) => Self {
+                Value: v as u128,
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/types.rs:275]: Why do we map `I64` to `u128`?

File: bindings/rust/wasmedge-sys/src/types.rs [hunk 3]
```diff
@@ -219,3 +221,146 @@ impl fmt::Display for ExternType {
...
+                Type: wasmedge::WasmEdge_ValType_I64,
+            },
+            Value::F32(v) => Self {
+                Value: v.to_bits() as u128,
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/types.rs:279]: Why do we map `F32` to `u128`?

File: bindings/rust/wasmedge-sys/src/types.rs [hunk 4]
```diff
@@ -219,3 +221,146 @@ impl fmt::Display for ExternType {
...
+                Type: wasmedge::WasmEdge_ValType_F32,
+            },
+            Value::F64(v) => Self {
+                Value: v.to_bits() as u128,
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/types.rs:283]: Why do we map `F64` to `u128`?

File: bindings/rust/wasmedge-sys/src/lib.rs [hunk 5]
```diff
@@ -1,46 +1,94 @@
...
 pub mod compiler;
+#[doc(hidden)]
 pub mod config;
+#[doc(hidden)]
 pub mod error;
+#[doc(hidden)]
 pub mod executor;
+#[doc(hidden)]
 pub mod import_obj;
+// #[doc(hidden)]
```
Comment (hydai) [inline bindings/rust/wasmedge-sys/src/lib.rs:31]: Why do we comment this out?

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by apepkuss** [comment]
Hi @hydai, the review issues have been fixed, please help review the latest changes again. Thanks a lot!

**Comment by hydai** [comment]
@apepkuss 
There are some issues are not resolved. Please take a look. Thanks.

**Comment by apepkuss** [comment]
@hydai, thanks for your reminder. I've already resolved the comments. Thanks!

**Review by hydai — APPROVED** [review]

**Pull request merged**